patches/chrome-browser-ui-page_info-page_info_ui.cc.patch

```diff
@@ -10,13 +10,13 @@ index de69865bea880d8dd1030e69d7ed28b34e450295..0031cbdd7c2ce79f14c0f9af59ca5feb
  #include "chrome/app/vector_icons/vector_icons.h"
  #include "chrome/browser/profiles/profile.h"
  #include "chrome/common/pref_names.h"
-@@ -604,6 +605,9 @@ const gfx::ImageSkia PageInfoUI::GetPermissionIcon(const PermissionInfo& info,
-     case CONTENT_SETTINGS_TYPE_USB_GUARD:
+@@ -616,6 +617,9 @@ const gfx::ImageSkia PageInfoUI::GetPermissionIcon(const PermissionInfo& info,
+     case ContentSettingsType::USB_GUARD:
        icon = &vector_icons::kUsbIcon;
        break;
-+    case CONTENT_SETTINGS_TYPE_AUTOPLAY:
++    case ContentSettingsType::AUTOPLAY:
```

Comment:
please convert to a `define` if you have to change the patch - can be done in a follow-up